### PR TITLE
[Term Entry] Python math-module math.asin()

### LINKED ISSUE #5078: [Topic Entry] Python math-module math.asin()
### Reason/inspiration (optional)

We would like a new term entry on the `math.asin()` method of the [`math-module`](https://www.codecademy.com/resources/docs/python/math-module) concept in Python. The entry should go in a new file under `docs/content/python/concepts/math-module/terms/math-asin/math-asin.md`

The entry should include the following:

- A description of the term
- A ##Syntax section that includes the syntax of the method and its details
- An ##Example section that includes a ```py ``` codeblock and ```shell ``` that shows the example output.
- A ##Codebyte Example to show the working of the method

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Topic Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by ahmadalsaadi89**
### Description

The Codecademy Docs Python Math Module required a new entry on the inverse function math.asin(x). A new markdown file math-asin.md was created in a new math-asin folder to go along with the other terms of the math-module.

The motivation behind contributing to Codecademy Docs was to utilize my new skillset in using Git and GitHub as directed by the Codecademy Git & GitHub course. This was initiated by a specific task to do so within the course.

I have made the entry very similar to the already existing math.sin(x) to enable less cognitive overload for users of Codecademy Docs when looking at both the function and its inverse.
 

### Issue Solved

 [Topic Entry] Python math-module math.asin() Issue-number: #5078 
Closes #5078 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/python/concepts/math-module/terms/math-asin/math-asin.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+Title: 'math-asin()'
+Description: 'Returns the arcsine or angle measured in radians between -pi/2 and pi/2 of input values ranging between -1 and 1.'
```
Comment (SaviDahegaonkar) [inline content/python/concepts/math-module/terms/math-asin/math-asin.md:3]:
``` suggestion
Description: 'Returns the arcsine(inverse sine) which is the angle measured in radians between -pi/2 and pi/2 of input values ranging between -1 and 1.'
```
Just did some small text adjustments to make the text look better.

**Review by SaviDahegaonkar — APPROVED** [review]
Hey @ahmadalsaadi89,
The entry is very nicely written. I had a look at your file and I feel now it is ready for a second round of review. So I hereby approve this PR.

File: content/python/concepts/math-module/terms/math-asin/math-asin.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  - 'Modules'
+  - 'Trigonometry'
+  - 'Functions'
+  - 'Arguments'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+Returns the **arcsine** or angle measured in radians of input values ranging between -1 and 1.
```
Comment (avdhoottt) [inline content/python/concepts/math-module/terms/math-asin/math-asin.md:18]: Write a longer description of 1-2 lines.

File: content/python/concepts/math-module/terms/math-asin/math-asin.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
+---
+Title: 'math-asin()'
```
Comment (avdhoottt) [inline content/python/concepts/math-module/terms/math-asin/math-asin.md:2]:
```suggestion
Title: '.asin()'
```

File: content/python/concepts/math-module/terms/math-asin/math-asin.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+---
+
+Returns the **arcsine** or angle measured in radians of input values ranging between -1 and 1.
+
+## Syntax
+
+```py
+math.asin(x)
+```
+
```
Comment (avdhoottt) [inline content/python/concepts/math-module/terms/math-asin/math-asin.md:25]: Explain the parameter `x` here in one line.

File: content/python/concepts/math-module/terms/math-asin/math-asin.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+## Example 1
+
+Use `math.asin()` to return arcsine of `1`:
```
Comment (avdhoottt) [inline content/python/concepts/math-module/terms/math-asin/math-asin.md:28]:
```suggestion
In the following example, the `math.asin()` function is used to return the arcsine of `1`:
```

File: content/python/concepts/math-module/terms/math-asin/math-asin.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+Returns the **arcsine** or angle measured in radians of input values ranging between -1 and 1.
+
+## Syntax
+
+```py
```
Comment (avdhoottt) [inline content/python/concepts/math-module/terms/math-asin/math-asin.md:22]:
```suggestion
```pseudo
```
The syntax is wrapped inside the pseudo block.

File: content/python/concepts/math-module/terms/math-asin/math-asin.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+
+The above code gives the following output measured in radians:
+
+```shell
+1.5707963267948966
+```
+
+## Example 2
+
+Use `math.asin()` to return arcsine of `-0.5`:
```
Comment (avdhoottt) [inline content/python/concepts/math-module/terms/math-asin/math-asin.md:44]:
```suggestion
In the following code block, the `math.asin()` is used to return the arcsine of `-0.5`:
```

**Review by SaviDahegaonkar — COMMENTED** [review]
Hi @avdhoottt , 
Can you please have a look at this PR on priority?

**Comment by SaviDahegaonkar** [comment]
> Hey @avdhoottt thank you for the suggestions. I have made the changes. I have committed all changes via the `add suggestion to batch` buttons. Many thanks!

Sure we will get back to you soon.

**Review by avdhoottt — APPROVED** [review]
LGTM!!

**Pull request merged**